Resources/Locale/en-US/guidebook/guides.ftl

```diff
@@ -20,3 +20,8 @@ guide-entry-ape = A.P.E.
 guide-entry-xenoarchaeology = Xenoarchaeology
 guide-entry-artifact-reports = Artifact Reports
 guide-entry-traversal-distorter = Traversal Distorter
+<<<<<<< HEAD
+guide-entry-chemistry = Chemistry
+=======
+guide-entry-chemistry = Chemistry
+>>>>>>> ac73b1ab677079ccda328ec66ab6cc65f171ff5f
```

Comment:
Those are git merge conflict markers. What are those doing here?

@PowerfulAtom111 Please learn this crucial part of git. https://stackoverflow.com/a/163659

Reply:
outdated but the issue still persists. i will figure this out

Reply:
it has been figured out 99% by Torun 1% by my dog. the changes have been resolved but github dont know nobody!